index.html

```diff
@@ -0,0 +1,347 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="assets/styles/style.css">
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+</head>
+
+<body>
+    <section class="showcase">
+        <header class="main-header">
+            <nav class="">
+                <a class="logo" href="/">
+                    <h1>Current Logo</h1>
+                </a>
+                <div class="">
+                    <i class='bx bx-menu'></i>
+                </div>
+            </nav>
+        </header>
+
+        <div class="showcase-description">
+            <h1>
+                I’m Djibril. <br>
+                Glad to see you!
+            </h1>
+
+            <p>
+                I’m a software developer! I can help you build a product , feature or website Look through some of my
+                work and experience! If you like what you see and have a project you need coded, don’t hesitate to
+                contact me.
+            </p>
+        </div>
+
+        <div class="showcase-contact">
+            <a href="" class="contact-link">LET'S CONNECT</a>
```

Comment:
- [ ] As this consider as the subtitle it shouldn't be the `a` tag but the `hx` tag, kindly change this to the `hx` tag for semantic reasons